docs/architecture.md

```diff
@@ -8,37 +8,47 @@ OSV runs on Google Cloud Platform, with the following main components:
 
 ## Cloud Datastore
 
-All vulnerability data is stored in Cloud Datastore, with the models defined
+All vulnerability data is stored in [Cloud Datastore], with the [models] defined
 [here].
 
+[Cloud Datastore]: https://cloud.google.com/datastore
+[models]:
+https://cloudplatform.googleblog.com/2015/08/Introduction-to-data-models-in-Cloud-Datastore.html
```

Comment:
This is very outdated and uses a different client library. Maybe link to https://googleapis.dev/python/python-ndb/latest/index.html#defining-entities-keys-and-properties instead?

Reply:
Yeah that's a good point, thanks. Updated in bc7de6a